index.d.ts

```diff
@@ -145,18 +145,23 @@ declare module '@primer/components' {
 
   export const StyledOcticon: React.FunctionComponent<StyledOcticonProps>
 
-  export interface DropdownProps extends React.Props<any>, StyledSystem.ColorProps, StyledSystem.SpaceProps, ButtonProps {
-    as?: React.ReactType
-    title?: string | React.ReactNode
-  }
+  export interface DropdownProps extends CommonProps, Omit<React.HTMLAttributes, 'color'> {}
```

Comment:
Probably want to extend `React.DetailsHTMLAttributes<HTMLElement>` here just to be more explicit.

Reply:
@clayplumridge dumb question, but are there docs somewhere on all the different types possible & naming for the `React.-` types?

Reply:
I'm not sure... I usually just declare a `<summary />` (or similar) in a `.tsx` file, right click and hit `Go to definition` in VS Code. It'll take you to `index.d.ts` for React that'll tell you what every built-in tag takes as props.